shinano-l.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" ?>
+<manifest>
+
+  <include name="base-l-aosp.xml"/>
+  <remote name="sony" fetch="git://github.com/sonyxperiadev/" />
+
+  <default remote="caf" revision="refs/tags/android-5.1.1_r1" sync-j="4"/>
+
+  <!-- Platform common things -->
+  <project name="device/generic/armv7-a-neon" path="device/generic/armv7-a-neon"/>
+  <project name="platform/external/libxml2" path="external/libxml2"/>
+  <project name="platform/external/wpa_supplicant_8" path="external/wpa_supplicant_8"/>
+  <project name="platform/hardware/broadcom/wlan" path="hardware/broadcom/wlan"/>
+  <project name="platform/hardware/qcom/audio" path="hardware/qcom/audio"/>
+  <project name="platform/hardware/qcom/gps" path="hardware/qcom/gps"/>
+  <project name="platform/hardware/qcom/media" path="hardware/qcom/media"/>
+  <project name="platform/hardware/qcom/wlan" path="hardware/qcom/wlan"/>
+  <project name="platform/system/qcom" path="system/qcom" revision="l_master"/>
+  <project name="platform/hardware/qcom/msm8x74" path="hardware/qcom/msm8x74"/>
+  <project name="platform/hardware/ril" path="hardware/ril"/>
+
+  <!--
+  <project name="platform/hardware/qcom/camera" path="hardware/qcom/camera" revision="l_LNX.LA.3.6"/>
+  -->
+  <project name="hardware_qcom_display" path="hardware/qcom/display" remote="b2g" revision="b2g-5.1.0_r1"/>
+  <project name="init_sh" path="external/init_sh" remote="b2g" revision="master"/>
+  <project name="platform_system_nfcd" path="system/nfcd" remote="b2g" revision="master"/>
+  <project name="timekeep" path="external/timekeep" remote="b2g" revision="master"/>
+
+  <project name="codeaurora_kernel_msm" path="kernel" remote="b2g" revision="aries-l"/>
+
+  <!-- Shinano/Arise specific things -->
+  <remove-project name="platform/hardware/libhardware"/>
+  <project name="platform_hardware_libhardware" path="hardware/libhardware" remote="b2g" revision="aries-l"/>
+  <project name="platform_external_libnfc-nci" path="external/libnfc-nci" remote="b2g" revision="aries-l"/>
+  <project name="device-sony-aries" path="device/sony/aries" groups="device" remote="b2g" revision="aries-l"/>
+  <project name="device-sony-leo" path="device/sony/leo" groups="device" remote="b2g" revision="aries-l"/>
+  <project name="vendor-sony-kernel" path="vendor/sony/kernel" groups="device" remote="b2g" revision="aries-l"/>
+
+  <project name="device-sony-shinano" path="device/sony/shinano" groups="device" remote="b2g" revision="aries-l"/>
+  <project name="thermanager" path="vendor/sony/system/thermanager" groups="device" remote="sony" revision="master"/>
```

Comment:
Maybe for those repos we should have well-known revision rather than master branch ? If there are incompatible upstream changes it might break stuff.